src/Tochka.JsonRpc.Client/Models/IJsonRpcCallContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using JetBrains.Annotations;
+using Tochka.JsonRpc.Common.Models.Request.Untyped;
+using Tochka.JsonRpc.Common.Models.Response;
+using Tochka.JsonRpc.Common.Models.Response.Errors;
+using Tochka.JsonRpc.Common.Models.Response.Untyped;
+
+namespace Tochka.JsonRpc.Client.Models;
+
+[PublicAPI]
+[SuppressMessage("Naming", "CA1716:Identifiers should not match keywords", Justification = "Error is official name")]
+public interface IJsonRpcCallContext
```

Comment:
doc?

Reply:
will be added later with all other documentation